src/method_ilan.jl

```diff
@@ -201,13 +246,22 @@ function ilan(
     end
```

Comment:
The lines
```
        Qp=Q;   Q=Qn;
        Qn=zero(Qn);
```
allocate quite a bit of memory. Don't you want to do inplace copy?  Smth like
```
 Qp[:]=Q;
 Q[:]=Qn; 
 Q[:]=0;
```